Musico/app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,30 @@
+class AuthenticationController < ApplicationController
```

Comment:
@mohseengs can use name as session controller new for render the form, create for login and destroy for logout. Only try to use rest full methods only